docs/collecting-data/collecting-from-own-applications/javascript-trackers/browser-tracker/browser-tracker-v3-reference/plugins/youtube-tracking/index.md

```diff
@@ -28,39 +37,51 @@ enableYouTubeTracking(/* options */);
 
 ## Quick Start
 
-To start tracking a YouTube video with default settings, use the snippet below:
+The snippets below show how to get started with the plugin, after [setting up your tracker](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/browser-tracker/browser-tracker-v3-reference/tracker-setup/installing-the-tracker-from-npm/index.md):
 
-**`index.html`**
+<Tabs>
+  <TabItem value="iframe" label="Iframe" default>
 
 ```html
-<html>
-  <head>
-    <title>Snowplow YouTube Tracking Example</title>
-    <script src="main.js"></script> 
-  </head>
-  <body>
-    <iframe
-      id="example-id"
-      src="https://www.youtube.com/embed/zSM4ZyVe8xs"
-    ></iframe>  
-  </body>
-</html>
+<iframe
+  id="yt-player"
+  src="https://www.youtube.com/embed/zSM4ZyVe8xs"
+></iframe>
 ```
 
-**`main.js`**
+```javascript
+import { enableYouTubeTracking } from '@snowplow/browser-plugin-youtube-tracking'
+
+enableYouTubeTracking({
+  id: 'yt-player'
+})
+```
+
+</TabItem>
```

Comment:
That's a great thing to add on the docs, nice!

Going through the plugin page after the changes seems a bit confusing though. This is because as a reader I cannot understand the difference just by seeing the titles `Iframe` and `YT.Player Instance`. 

What I would expect is either:
1) Stating in the code directly that the `id` attribute accepts an existing player instance as well e.g.
```ts
import { enableYouTubeTracking } from '@snowplow/browser-plugin-youtube-tracking';

const videoId =  'zSM4ZyVe8xs';

enableYouTubeTracking({
  id: videoId,
});

/* The `id` attribute can also accept an existing YT.Player video instance */

const videoInstance = new YT.Player('yt-player', {
 videoId
});

enableYouTubeTracking({
  id: videoInstance,
});
```
 
 2) Add an info note with the same comment.

Reply:
That makes sense, I think I like the note idea more, so I've put in a note explaining what `id` accepts before the examples.